telegram/internal/upconv/upconv.go

```diff
@@ -93,7 +98,10 @@ func ShortChatMessage(u *tg.UpdateShortChatMessage) *tg.UpdateShort {
 // ShortSentMessage converts UpdateShortSentMessage to UpdateShort.
 func ShortSentMessage(u *tg.UpdateShortSentMessage) *tg.UpdateShort {
 	msg := &tg.Message{
-		Out:  u.Out,
+		// UpdateShortSentMessage is shortened constructor containing info
+		// on one **outgoing** message to a contact, so `Message.Out` should
+		// always be true
+		Out:  true,
```

Comment:
Why we should set it explicitly if u.Out is always true?

Reply:
I am a little confused here, if we respects the defintion of **UpdateShortSentMessage**, so msg.Out should set to true explicitly, and then field **UpdateShortSentMessage.Out** will be useless.
If we using u.Out as msg.Out directly, but as you can see [in the documentation](https://core.telegram.org/constructor/updateShortSentMessage), u.Out is an **optional** field, that is mean it might be not set?
So should i update this to
```Golang
msg := &tg.Message{
		ID:   u.ID,
		Date: u.Date,
}
msg.SetOut(u.Out)
```
?

Reply:
Yes, something like that.